RecoTauTag/RecoTau/plugins/PFRecoTauTagInfoProducer.cc

```diff
@@ -98,7 +98,8 @@ else{
   
   auto resultExt = std::make_unique<PFTauTagInfoCollection>();  
   for(JetTracksAssociationCollection::const_iterator iAssoc=thePFJetTracksAssociatorCollection->begin();iAssoc!=thePFJetTracksAssociatorCollection->end();iAssoc++){
-    PFTauTagInfo myPFTauTagInfo=PFRecoTauTagInfoAlgo_->buildPFTauTagInfo((*iAssoc).first.castTo<PFJetRef>(),thePFCandsInTheEvent,(*iAssoc).second,thePV);
+    // PFTauTagInfo myPFTauTagInfo=PFRecoTauTagInfoAlgo_->buildPFTauTagInfo((*iAssoc).first.castTo<JetBaseRef>(),thePFCandsInTheEvent,(*iAssoc).second,thePV);
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant